emmanuelmacharia/Store-Manager
# Development

## What does this PR do? ##
It has all the endpoints, tests and authentication. It also resolves validation of user input.

## Description of tasks to be done ##
Create endpoints for the store manager application for getting and creating all products and sales; gives authentication for both where the user has to be logged in and gets an access token.

## How to test it ##
* clone this repository
* Ensure you have all requirements installed (check the requirements.txt file in `Store-Manager/requirements.txt`).
* run flask using  `FLASK_APP=run.py` then `flask run` commands
* Ensure you have Postman installed, and paste the URL given when the app runs into Postman
* enter the Urls as specified in my `app/api/v1/__init__.py` file (ensure the url prefix `api/v1`is present) eg. `api/v1/register`
* go ahead and enter the information required as specified

## PT stories ##
https://www.pivotaltracker.com/n/projects/2203951
## Screenshots ##

  
![screenshot 46](https://user-images.githubusercontent.com/33549531/47409536-b6042e80-d76b-11e8-85bd-60d3986620bd.png)
